<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PoissonDialog</class>
 <widget class="QDialog" name="PoissonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Poisson reconstruction</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Min triangle angle:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="m_inputAngle">
     <property name="suffix">
      <string> Â°</string>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>30.000000000000000</double>
     </property>
     <property name="value">
      <double>20.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Max triangle size:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="m_inputRadius">
     <property name="suffix">
      <string> * average spacing</string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Approximation error:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="m_inputDistance">
     <property name="suffix">
      <string> * average spacing</string>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.250000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Solver:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="m_inputSolver">
     <property name="toolTip">
      <string extracomment="Name of the sparse solver"/>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="m_inputTwoPasses">
     <property name="text">
      <string>Uses two passes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PoissonDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PoissonDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
